backend/src/nodes/impl/cas.py

```diff
@@ -35,9 +35,9 @@ def create_cas_mask(img: np.ndarray, kernel, bias: float = 2) -> np.ndarray:
 
     l = _luminance(img)
     min_l = cv2.erode(l, kernel)
-    max_l = cv2.dilate(l, kernel, dst=l)
+    max_l = cv2.dilate(l, kernel)
     min_d = np.minimum(1.0 - max_l, min_l, out=min_l)  # type: ignore
-    max_l += 1e-8  # type: ignore
+    max_l = np.add(max_l, 1e-8)
```

Comment:
This can be reverted.

```suggestion
    max_l += 1e-8 # type: ignore
```